packages/ui-scripts/lib/publish.js

```diff
@@ -25,43 +25,40 @@ const { getPackageJSON } = require('@instructure/pkg-utils')
 const { error, info } = require('@instructure/command-utils')
 
 const { publishPackages, createNPMRCFile } = require('./utils/npm')
-const { checkIfGitTagExists, isReleaseCommit } = require('./utils/git')
-const { setupGit, checkWorkingDirectory } = require('./utils/git')
+const { isReleaseCommit, setupGit } = require('./utils/git')
 const { getConfig } = require('./utils/config')
 
 try {
   const pkgJSON = getPackageJSON()
-  // optional version argument:
-  // e.g. ui-scripts --publish alpha
-  publish(pkgJSON.name, pkgJSON.version, process.argv[3], getConfig(pkgJSON))
+
+  publish({
+    packageName: pkgJSON.name,
+    currentVersion: pkgJSON.version,
+    packageConfig: getConfig(pkgJSON)
+  })
 } catch (err) {
   error(err)
   process.exit(1)
 }
 
-async function publish(packageName, currentVersion, preidAndTag, config = {}) {
+async function publish(options) {
+  const { packageName, currentVersion, packageConfig } = options
+  //Set up git user.name and user.email
   setupGit()
-  createNPMRCFile(config)
-  checkWorkingDirectory()
```

Comment:
is this not needed?

Reply:
It does (or did) virtually nothing. Checks if the working directory is clean or not, but there is no possible way I can see where it would't.